pages/events.js

```diff
@@ -13,31 +13,39 @@ const useStyles = makeStyles(() => ({
 const Events = ({ pastEvents, upcomingEvents }) => {
   const classes = useStyles();
 
+  const [events, setEvents] = useState({ pastEvents, upcomingEvents });
+
+  useEffect(async () => {
+    setEvents(await fetchEvents());
+  }, [setEvents, fetchEvents]);
+
   return (
     <Box m={4} className={classes.root}>
       <EventsHeadline />
-      {false && <Box>filters</Box>}
       <h1>Již brzy</h1>
-      {isEmpty(upcomingEvents) && <img src={'/images/gugtravolta.gif'} />}
+      {isEmpty(events.upcomingEvents) && <img src={'/images/gugtravolta.gif'} />}
```

Comment:
What isEmpty? Cannot be used simple: events?.upcomingEvents 
or
!!(events.upcomingEvents) ? travolta.gif : ''

Reply:
Upcoming events is array. upcomingEvents is always populated at least by empty array and Boolean([]) returns true

Reply:
Díky!